package.json

```
@@ -19,20 +19,20 @@
     "tsc:validate": "tsc --noEmit --incremental false"
   },
   "dependencies": {
-    "@tanstack/react-query": "^5.0.0",
+    "@tanstack/react-query": "^5.0.5",
     "cookies-next": "^4.0.0",
     "date-fns": "^2.30.0",
     "formik": "^2.4.5",
-    "i18next": "^23.5.1",
+    "i18next": "^23.6.0",
     "markdown-to-jsx": "^7.3.2",
-    "next": "13.5.5",
+    "next": "13.5.6",
```

```suggestion
    "next": "^13.5.6",
```

(to match all the others)

---

This is what `package-lock.json` is for.. if the docker build isn't picking up the correct version from `package-lock.json`, then the docker build is misconfigured.

---

We need to install `next` package only to be able to run the application in the final build. I've tested many scenario with @ken-blanchard and this is the only way we could make sure the final build had the proper next version when starting the container.

https://stackoverflow.com/a/45566871

---

I've added a comment in the package.json for next version.
https://github.com/DTS-STN/passport-status/pull/538/commits/6cd5e192ef3c8091f5f406bed14635c96af25d75

---

If the intention was to ONLY install next, this command does not achieve the expected results. It installs all dependencies in package.json

`npm install --no-save next `

In other projects I've reverted to use `npm ci --omit=dev`. The command `npm ci` respects the lock file.

---

AFAIK There is no clean way to install a single dependency from your package.json using npm while having it respect package-lock.json.

---

Will do the same as you've suggested.

---

Fixed